.babelrc

```diff
@@ -1,5 +1,8 @@
 {
-  "presets": [
-    ["env", { "modules": false } ]
```

Comment:
any idea why we had modules: false to begin with?

Reply:
https://github.com/rails/webpacker/blob/master/lib/install/config/.babelrc#L4 is the only reason I think.

(Mostly our webpack config is just webpacker default.)